war/src/main/less/abstracts/theme.less

```diff
@@ -66,13 +66,6 @@
   --header-link-bg-classic-hover: #404040;
   --header-link-bg-classic-active: #404040;
 
-  // Header new UI colors
-  --header-bg-v2: var(--primary-hover);
```

Comment:
This is is used in the dark theme plugin, it would make sense to remove it there as well then, the other removals of this block are commented out in the dark theme and solarized plugin or unused.